packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -17,6 +20,8 @@ const props = withDefaults(defineProps<OnyxInputProps>(), {
   autocapitalize: "sentences",
   readonly: false,
   loading: false,
+  showSuccessIcon: true,
+  showClearIcon: true,
```

Comment:
We agreed to not default booleans to true so we should e.g. rename the properties to `hideSuccessIcon` and `hideClearIcon`, see https://onyx.schwarz/principles/technical-vision.html#component-interface (point 6)

Reply:
changed it